frontend/src/app/main/data/dashboard.cljs

```diff
@@ -999,8 +1000,10 @@
      (watch [_ state _]
        (let [team-id (:current-team-id state)]
          (if (empty? term)
-           (rx/of (rt/nav :dashboard-search
-                          {:team-id team-id}))
+           (do
```

Comment:
It looks great but I think it would be better moving that code to an effect event and add something like:
```
     ptk/EffectEvent
     (effect [_ _ _]
       (dom/focus! (dom/get-element "search-input"))
```